### PR TITLE
Add 'Internet TV AG' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22onlinetvrecorder%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22entertainment%22%5D%2C%22name%22%3A%22Internet%20TV%20AG%22%2C%22runs%22%3A%5B%22onlinetvrecorder.com%22%5D%2C%22address%22%3A%22306%20Victoria%20House%5Cn361%20Victoria%5CnSeychelles%5Cn%5Cn%22%2C%22phone%22%3A%22%2B1%20510%20984%205218%22%2C%22email%22%3A%22webmaster%40onlinetvrecorder.com%22%2C%22web%22%3A%22http%3A%2F%2Fwww.onlinetvrecorder.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.onlinetvrecorder.com%2Fv2%2Fcontact%2Fgdpr%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1350%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22comments%22%3A%5B%22Achtung%2C%20teure%20Auslandsrufnummer!%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Comment by WebworkrNet** [comment]
I have opened this issue via [datenanfragen.de](https://www.datenanfragen.de).
You can contact me if you have any questions.

File: companies/onlinetvrecorder.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Internet TV AG",
+    "runs": [
+        "onlinetvrecorder.com"
+    ],
+    "address": "306 Victoria House\n361 Victoria\nSeychelles\n\n",
```
Comment (moriarty-jim) [inline companies/onlinetvrecorder.json:13]:
```suggestion
    "address": "306 Victoria House\nVictoria Mahe\nSeychelles",
```
fixed formatting; see [https://www.onlinetvrecorder.com/v2/contact/gdpr](https://www.onlinetvrecorder.com/v2/contact/gdpr)

File: companies/onlinetvrecorder.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "onlinetvrecorder",
+    "relevant-countries": [
+        "de"
```
Comment (moriarty-jim) [inline companies/onlinetvrecorder.json:4]:
```suggestion
        "all"
```
I would suggest "all" as the service is available worldwide and in multiple languages.

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
fixed address and suggestion for `relevant-countries` (optional)

**Review by baltpeter — COMMENTED** [review]
Thanks, everyone!

**Review by baltpeter — APPROVED** [review]

**Pull request merged**